packages/grid_client/src/high_level/twinDeploymentHandler.ts

```diff
@@ -265,6 +265,30 @@ class TwinDeploymentHandler {
     return deployments;
   }
 
+  async checkFarmIps(twinDeployments: TwinDeployment[]) {
+    const farms: { [key: string]: { ips: number } } = {};
+    for (const twinDeployment of twinDeployments) {
```

Comment:
not all deployments should be included only deployments with the `Deploy` operation.